object_tracker_0/tests/test_inference_metrics.py

```diff
@@ -0,0 +1,76 @@
+import pytest
```

Comment:
This file is unrelated, but realized that I missed it in another PR so just piggybacking it here